mastracode/factory-ui/AGENTS.md

```
@@ -1,12 +1,12 @@
-Web UI (Vite against the web host API): `pnpm --filter ./mastracode/factory-ui web`
+Dev (API on :4111 + Vite on :5173): `pnpm --filter ./mastracode/factory-ui web`
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)